src/backend/OpenCL/CHIPBackendOpenCL.cc

```diff
@@ -407,6 +407,9 @@ CHIPDeviceOpenCL::CHIPDeviceOpenCL(CHIPContextOpenCL *ChipCtx,
       *DevIn, CL_DEVICE_SINGLE_FP_ATOMIC_CAPABILITIES_EXT);
   Fp64AtomicAddCapabilities_ = getFPAtomicCapabilities(
       *DevIn, CL_DEVICE_DOUBLE_FP_ATOMIC_CAPABILITIES_EXT);
+
+  auto DevExts = DevIn->getInfo<CL_DEVICE_EXTENSIONS>();
+  HasSubgroupBallot_ = DevExts.find("cl_khr_subgroup_ballot");
```

Comment:
I think the LLVM style doesn't add underscores for member variables, or does it?

Reply:
Underscores in the names are probably discouraged in LLVM's coding standard. That said LLVM has them in some uncommon cases - for example:
```
  LazyValueInfo(AssumptionCache *AC_, const DataLayout *DL_,
                TargetLibraryInfo *TLI_)
      : AC(AC_), DL(DL_), TLI(TLI_) {}
```
It is Paulius' request that private and protected members have `_` suffix in the names. (There is also a [clang-tidy configuration](https://github.com/CHIP-SPV/chipStar/blob/d31f90490d6f9ff7ffc85979e792c4c6467c7895/.clang-tidy#L13-L16) for this too).